internal/metrics/metrics.go

```diff
@@ -182,6 +193,11 @@ func (m *Metrics) RecordInvExecutionTime(h heuristic.Heuristic, latency float64)
 	m.InvExecutionTime.WithLabelValues(ht).Set(latency)
 }
 
+// IncMissedBlock ... Increments the number of missed blocks
+func (m *Metrics) IncMissedBlock(pUUID core.PUUID) {
+	m.MissedBlocks.WithLabelValues(pUUID.String()).Inc()
+}
+
 // IncActiveHeuristics ... Increments the number of active heuristics
```

Comment:
Don't we also want the network that we missed blocks for? The puuid is just a string uuid for each pipeline, which means we can't charts split by network

Reply:
True but the network byte enum is also encoded in the PUUID; meaning that it's already being presented. 
https://base-org.github.io/pessimism/architecture/etl

Reply:
Yes, I understand that but my point is that its encoded into the uuid so when we try to use this in our monitoring dashboard, we will only be able to tag missed blocks by pipeline and not by network. If this is the behavior we're looking to display though then this is fine